MesaService/ServiceInstaller.Designer.cs

```diff
@@ -0,0 +1,61 @@
+﻿
+namespace MesaService
+{
+    partial class ServiceInstaller
+    {
+        /// <summary>
+        /// Required designer variable.
+        /// </summary>
+        private System.ComponentModel.IContainer components = null;
+
+        /// <summary> 
+        /// Clean up any resources being used.
+        /// </summary>
+        /// <param name="disposing">true if managed resources should be disposed; otherwise, false.</param>
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && (components != null))
+            {
+                components.Dispose();
+            }
+            base.Dispose(disposing);
+        }
+
+        #region Component Designer generated code
+
+        /// <summary>
+        /// Required method for Designer support - do not modify
+        /// the contents of this method with the code editor.
+        /// </summary>
+        private void InitializeComponent()
+        {
+            this.serviceProcessInstaller1 = new System.ServiceProcess.ServiceProcessInstaller();
+            this.serviceInstaller1 = new System.ServiceProcess.ServiceInstaller();
+            // 
+            // serviceProcessInstaller1
+            // 
+            this.serviceProcessInstaller1.Account = System.ServiceProcess.ServiceAccount.LocalSystem;
+            this.serviceProcessInstaller1.Password = null;
+            this.serviceProcessInstaller1.Username = null;
+            // 
+            // serviceInstaller1
+            // 
+            this.serviceInstaller1.Description = "Performs background tasks for the MesaSuite software.";
+            this.serviceInstaller1.DisplayName = "MesaService";
+            this.serviceInstaller1.ServiceName = "MainService";
+            this.serviceInstaller1.StartType = System.ServiceProcess.ServiceStartMode.Automatic;
+            // 
+            // ServiceInstaller
+            // 
+            this.Installers.AddRange(new System.Configuration.Install.Installer[] {

```

Comment:
Line spacing on this is kinda weird - maybe tab the member vars of the list up a level so they're not on the same?

Reply:
This is an auto-generated file.  Any changes I make to this file would just be overwritten the moment I make a change in the designer.